src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -548,7 +548,8 @@ private boolean parseOption(String key, final String stringValue, final boolean
             } else {
                 value = constructFromString(getUnderlyingType(optionDefinition.field), stringValue);
             }
-
+            // TODO - Picard legacy parsing is not taking into account range within options
+            // TODO - should we add support here?
```

Comment:
Yes, sounds like a reasonable place (please include the name of the argument int he message here as well).